domains/li0ard.is-cool.dev.json

```diff
@@ -0,0 +1,14 @@
+{
+    "domain": "is-cool.dev",
+    "subdomain": "li0ard",
+
+    "owner": {
+        "email": "li0ard@proton.me"
+    },
+
+    "record": {
+        "CNAME": "li0ard.github.io"
```

Comment:
CNAME redirects to another domain which is then redirected because it is invalid. Please provide actual content and not multi layer of redirects.

Reply:
Sorry. I fixed this.